packages/components/pie-input/src/index.ts

```diff
@@ -218,12 +218,15 @@ export class PieInput extends FormControlMixin(RtlMixin(LitElement)) implements
                     placeholder=${ifDefined(placeholder)}
                     ?readonly=${readonly}
                     ?required=${required}
+                    aria-describedby=${ifDefined(assistiveText ? 'assistive-text' : undefined)}
```

Comment:
would `nothing` work here instead of undefined? I often forget the difference 😂 Does `undefined` mean the attribute won't be added at all?